spec/support/libraryStubData.js

```diff
@@ -126,7 +145,22 @@ var libraryStubData = {
             return (new Date()).getTime();
         }
     },
-    'utilities.js': {},
+    'utilities.js': {
+        isArray: function(array){
+            return Array.isArray(array);
+        },
+        isNumber: function(value){
+            return !Number.isNaN(value);
+        },
+        isString: function(string){
+            if (typeof myVar === 'string' || myVar instanceof String){
```

Comment:
could simplify to just `return typeof myVar === 'string' || myVar instanceof String;` and skip the if/else